storage/filestore.go

```diff
@@ -221,7 +221,7 @@ func (f *FilesystemStore) Set(name string, meta []byte) error {
 	os.RemoveAll(fp)
 
 	// Write the file to disk
-	if err = ioutil.WriteFile(fp, meta, f.perms); err != nil {
+	if err = ioutil.WriteFile(fp, meta, f.perms&notary.PermExecutableMask); err != nil {
```

Comment:
my understanding was that `NewPrivateSimpleFileStore` would use the private perms pair and `NewFilesystemStore` would use the public ones.

We would have to change the usage of `NewFilesystemStore` to `NewPrivateSimpleFileStore` in `client.go` to keep the existing permissions behavior.